scripts/netContain/ReleaseContainer.sh

```diff
@@ -0,0 +1,15 @@
+if [ "$1" = "" ]; then
+  echo "Please provide the contiv release version to use"
+  echo "Usage: ./ReleaseContainer.sh <contiv version>"
+  echo "Example: ./ReleaseContainer.sh v0.1-11-30-2016.20-08-20.UTC"
+  echo "Released versions are available from https://github.com/contiv/netplugin/releases"
+  exit 1
+else
+  contivVersion=$1
+fi
+echo "Login to docker hub as contiv user"
+docker login -u contiv
```

Comment:
If you have are logged in as a different user, you will have to use a different image name. As we decided to have contiv/netplugin, I did not make the user name configurable. If required, I can make a change to make both image name and user name configurable.